### ISSUE TITLE
3D Urban Environment

### ISSUE THREAD

**Issue by kamami** [mapbox/mapbox-maps-flutter#220]
When will this feature be available in this sdk?
https://www.mapbox.com/blog/standard-core-style
When will be a new update released in general or is this repo not maintained anymore?

**Comment by yunikkk** [comment]
@kamami yes we plan to update the maps plugin to support v11 with the shiny Standard style later this year. 
Tickets reported in the repo will be addressed as well.
Sorry not many updates were done here last months, we were busy with other priorities and projects.

**Comment by kamami** [comment]
Is this eventually implemented?

**Comment by O-Hannonen** [comment]
Cleaning up some old issues here:
This has been part of `MapboxStyles.STANDARD` as stable feature since mapbox_maps_flutter `v2.9.0`. Closing this issue as completed.
